src/nav/alerts-applied-intelligence.yml

```diff
@@ -6,7 +6,7 @@ pages:
   - title: Alerting best practice guide
     path: /docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alerts-best-practices
   - title: Alerts & AI overview page
-    path: /docs/alerts-applied-intelligence/new-relic-alerts/get-started/
+    path: docs/alerts-applied-intelligence/new-relic-alerts/get-started/alerts-ai-overview-page
```

Comment:
@bradleycamacho is it a problem if you're missing that initial / in the path?

Reply:
ooop

Reply:
Fixed :finger-guns: